### DISCUSSION TITLE
TIFFImagePyramidExporter - crashes after creating level 0 and 1

### DISCUSSION THREAD

**Discussion by mhoibo** [FAST-Imaging/FAST#211] [General]
Hi, 
I am trying to extract TMA cores from a TMA image and save the cores as separate pyramidal tiff images. I have used the TMA extractor to extract the cores, and want to use TIFFImagePyramidExporter to create a TIFF pyramid image like this: 
```
fast.TIFFImagePyramidExporter.create(path/+ file_name + ".tiff").connect(core).run()
```
However, it crashes after level 0 and 1 are created (last output line is "Done creating level 1), giving me this error: 
`Quantization table 0x00 was not defined`
Looking at the the size at level 1, I think level 2 will be too small to create (L83 in ImagePyramid.cpp), and that it crashes after that, but I do not understand why.

I was also wondering if I do make this work, and I later want to get level 2 (x10) from this TMA core image, if that is possible in a convenient way even if the level does not exist? Other than reading the whole TMA image at this lower level and extracting the TMA cores again at a lower level.

Thank you in advance!

**Comment by smistad** [comment]
Hi @mhoibo 

I forgot to answer this. This was due to a bug introduced in 4.9.
Although not a complete fix, as I don't completely understand the problem but it is related to jpeg reading and writing of tiffs.
This quick-fix should work around it at least: https://github.com/smistad/FAST/commit/376d5f351312c5c6e2527d5214569a55ede020a1

**Reply by mhoibo** [reply]
Thank you!
